src/components/swapv2/SingleTokenInfo.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { Trans } from '@lingui/macro'
+import { Flex } from 'rebass'
+import Loader from 'components/Loader'
+import CurrencyLogo from 'components/CurrencyLogo'
+import { formattedNum } from 'utils'
+import { useRef } from 'react'
+import { formatDollarAmount } from 'utils/numbers'
+import { isMobile } from 'react-device-detect'
+import { TokenInfo } from 'hooks/useTokenInfo'
+import { Currency } from '@kyberswap/ks-sdk-core'
+
+const NOT_AVAIALBLE = '--'
+const NUM_LINE_DESC = 5
+// 2 styles: border and no border
+const Wrapper = styled.div<{ borderBottom?: boolean }>`
+  width: 100%;
+  padding: 0px
+  margin-top: 0px;
+  margin-bottom: 0px;
+  border: none;
+  ${({ borderBottom, theme }) =>
+    borderBottom
+      ? `
+  border-bottom: 1px solid ${theme.border}; 
+  margin-bottom: 30px;
+  padding-bottom: 30px;`
+      : ``}
+  ${({ theme, borderBottom }) => theme.mediaWidth.upToSmall`
+    margin-top: 24px;
+    ${
+      borderBottom
+        ? `
+    margin-bottom: 10px;
+    margin-padding: 10px;
+    `
+        : ``
+    }
+`}
+`
+
+const InfoRow = styled.div<{ isFirst?: boolean; isLast?: boolean }>`
+  width: 33%;
+  text-align: ${({ isLast }) => (isLast ? 'right' : `left`)};
+  padding: 7px 0px 7px ${({ isFirst }) => (isFirst ? '0px' : '40px')};
+  border-left: ${({ theme, isFirst }) => (isFirst ? 'none' : `1px solid ${theme.border}`)};
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    border: none;
+    padding: 20px 0px;
+`}
+`
+
+const InfoRowValue = styled.div`
+  color: ${({ theme }) => theme.text};
+  font-size: 20px;
+  font-weight: 400;
+`
+
+const InfoRowLabel = styled.div`
+  color: ${({ theme }) => theme.subText};
+  font-size: 14px;
+  font-weight: 500;
+  padding-bottom: 8px;
+`
+
+const AboutText = styled.div`
+  color: ${({ theme }) => theme.subText};
+  font-size: 20px;
+  font-weight: 500;
+  margin-left: 10px;
+  margin-bottom: 10px;
+`
+
+const LINE_HEIGHT = 24
+const DescText = styled(InfoRowLabel)<{ showLimitLine: boolean }>`
+  margin: 10px 0px;
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    margin: 10px 0px 0px 0px;
+  `}
+  p {
+    line-height: ${LINE_HEIGHT}px;
+    margin: 0;
+    ${({ showLimitLine }) =>
+      showLimitLine
+        ? `
+    text-overflow:ellipsis;
+    overflow:hidden;
+    display: -webkit-box !important;
+    height: ${LINE_HEIGHT * NUM_LINE_DESC}px;
+    -webkit-line-clamp: ${NUM_LINE_DESC};
+    -webkit-box-orient: vertical;
+    white-space: normal;
+  `
+        : ''}
+  }
+`
+const SeeMore = styled.a`
+  cursor: pointer;
+  margin-top: 5px;
+  display: block;
+  text-align: right;
+`
+
+function replaceHtml(text: string) {
```

Comment:
People can't understand when read this :(.

Reply:
<img width="685" alt="image" src="https://user-images.githubusercontent.com/33005392/174734609-01ca48ca-eda7-4017-996f-4ef2ebd64c38.png">
you need something like this.